<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="21"
           failures="4"
           errors="0"
           skipped="0"
           time="626.432">
  <properties />
            <testcase name="C_addtrunk" time="28.440" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest">
                <failure type="java.lang.AssertionError"
                  message="IAX1注册失败"
              >
<![CDATA[java.lang.AssertionError: IAX1注册失败
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertNotEquals(YsAssert.java:912)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.assertTrunkStatus(YS_Trunk.java:223)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.addTrunk(YS_Trunk.java:181)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest.C_addtrunk(BeforeTest.java:95)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="E_addtrunk" time="26.652" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest">
                <failure type="java.lang.AssertionError"
                  message="SPX1注册失败"
              >
<![CDATA[java.lang.AssertionError: SPX1注册失败
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertNotEquals(YsAssert.java:912)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.assertTrunkStatus(YS_Trunk.java:223)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.addTrunk(YS_Trunk.java:181)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest.E_addtrunk(BeforeTest.java:106)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="B_addtrunk" time="43.079" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest">
                <failure type="java.lang.AssertionError"
                  message="SIP1注册失败"
              >
<![CDATA[java.lang.AssertionError: SIP1注册失败
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertNotEquals(YsAssert.java:912)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.assertTrunkStatus(YS_Trunk.java:223)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.addTrunk(YS_Trunk.java:181)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest.B_addtrunk(BeforeTest.java:89)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="D_addtrunk" time="26.769" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest">
                <failure type="java.lang.AssertionError"
                  message="SPS1注册失败"
              >
<![CDATA[java.lang.AssertionError: SPS1注册失败
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertNotEquals(YsAssert.java:912)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.assertTrunkStatus(YS_Trunk.java:223)
  com.yeastar.swebtest.driver.YSMethod.YS_Trunk.addTrunk(YS_Trunk.java:181)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest.D_addtrunk(BeforeTest.java:100)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="G_addOutRoute6" time="13.670" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="L_setRecord" time="42.402" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute2" time="13.634" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute3" time="13.748" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="A_editExtenName" time="49.348" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="A_addExtension" time="49.062" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute8" time="13.732" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="H_addivr" time="46.159" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute7" time="13.687" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute1" time="31.184" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="A_addExtensionGroup" time="33.912" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="J_addqueue" time="36.237" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="K_addconference" time="35.274" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute5" time="13.638" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="I_addringgroup" time="30.979" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="F_addInRoute" time="51.023" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
                <testcase name="G_addOutRoute4" time="13.803" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.BeforeTest" />
      
</testsuite>
